railties/lib/rails/generators/rails/app/templates/.devcontainer/compose.yaml.tt

```diff
@@ -0,0 +1,56 @@
+services:
+  rails-app:
+    build:
+      context: ..
+      dockerfile: .devcontainer/Dockerfile
+
+    volumes:
+    - ../..:/workspaces:cached
+
+    # Overrides default command so things don't shut down after the process ends.
+    command: sleep infinity
+
+    networks:
+    - default
+
+    # Uncomment the next line to use a non-root user for all processes.
+    # user: vscode
+
+    # Use "forwardPorts" in **devcontainer.json** to forward an app port locally.
+    # (Adding the "ports" property to this file will not forward from a Codespace.)
+    ports:
+    - 3000:3000
```

Comment:
Do we need the 3000 port? VScode maps automatically as soon as you run `rails s`